src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
```suggestion
```

Reply:
It was made automatically by the php lsp (intelephense). Removed 👍🏻